client/views/admin/integrations/exampleIncomingData.js

```diff
@@ -10,7 +10,7 @@ export function useExampleData({ additionalFields, url }) {
 					title: 'Rocket.Chat',
 					title_link: 'https://rocket.chat',
 					text: 'Rocket.Chat, the best open source chat',
-					image_url: '/images/integration-attachment-example.png',
+					image_url: 'https://open.rocket.chat/images/logo/logo.svg',
```

Comment:
I have a few concerns:

- The load is probably negligible but still I don't think Open server should be the one serving those images. A better alternative would be github (any image from https://github.com/RocketChat/Rocket.Chat.Artwork) or another CDN where we have images hosted;
- we're creating an external dependency, which is also not ideal (airgapped environments will also not work).. so instead we could chose another internal image.. the server logo might be best choice, as it will probably never be deleted.